src/IIIFingest/client.py

```diff
@@ -112,27 +114,43 @@ def _get_manifest_url(self, manifest_name: str, prezi_version: int = 3) -> str:
         """Constructs the manifest URL."""
         return f"{self.manifest_base_url}{manifest_name}:MANIFEST:{prezi_version}"
 
-    def upload(self, images: List[dict], s3_path: str = "") -> List[Asset]:
+    def upload(
+        self, images: List[dict], s3_path: str = "", with_uuid=None
+    ) -> List[Asset]:
         """
         Uploads a list of images to the MPS ingest bucket in S3.
         Returns a list of assets.
+        image dict format
+        {
+            "id": "id123",
+            "label": "",
+            "filepath": ".../lts-iiif-ingest-service/tests/images/27.586.1-cm-2016-02-09.tif", # either filepath or fileobj is required
+            "fileobj": "", # either filepath or fileobj is required
+        }
```

Comment:
I like having this! Maybe a good idea to include the optional `asset_id` here too? I think that's the only optional parameter that does anything.

Reply:
Yeah, this would be a case where using a TypedDict might be helpful as we don't have / need an Image class.